DOC-303-Security-doc-conversion (#910)
spectrocloud/librarium: content/docs/11-security/3-tenant-cluster-security.md

## Kubernetes Runtime Security

Kubernetes runtime security support is done through a variety of add-on packages such as Sysdig Falco, Palo Alto Network Prisma Cloud (Twistlock). These packs are optional and allow users to bring their own license to use commercial solutions. <p></p><br />
Spectro Cloud's add-on pack integration is extensible. If a user prefers a particular security solution that is not provided by Spectro Cloud out-of-box, it can be easily added as long as the solution supports Helm Charts, Kustomize, or Operators.<p></p><br />

## Kubernetes Authentication and Authorization

Kubernetes cluster authentication can be optionally integrated with Kubelogin and with OIDC-based authentication/authorization against external IDP. This enables group membership based access control on different namespaces within the tenant Kubernetes cluster. Spectro Cloud's Terraform provider also supports automatically set role binding on namespaces with users or groups.<p></p><br />

## Kubernetes Conformance and Compliance Test

> Spectro Cloud allows users to run Kubernetes conformance and compliance tests, using Kubebench, Kubehunter, and Sonobuoy periodically based, on schedule. These tests ensure users that their tenant clusters are secure, compliant, conformant, and up to date.

Reviewer: ```suggestion
Spectro Cloud allows you to periodically run Kubernetes conformance and compliance tests using Kubebench, Kubehunter, and Sonobuoy based on a schedule you provide.
```
